$OpenBSD: patch-phoenix_pro,v 1.2 2011/03/02 06:26:04 jasper Exp $

Don't build everything twice, just once takes long enough already.

--- phoenix.pro.orig	Fri Feb 18 17:06:20 2011
+++ phoenix.pro	Thu Feb 24 19:02:01 2011
@@ -26,7 +26,7 @@
 
 
 
-CONFIG += debug_and_release
+CONFIG += release
 win32 {
 # release build using msvc 2010 needs to use Multi-threaded (/MT) for the code generation/runtime library option
 # release build using msvc 2010 needs to add msvcrt.lib;%(IgnoreSpecificDefaultLibraries) to the linker/no default libraries option
